irohad/ordering/impl/on_demand_ordering_gate.cpp

```diff
@@ -51,6 +51,7 @@ OnDemandOrderingGate::OnDemandOrderingGate(
             // notify our ordering service about new round
             ordering_service_->onCollaborationOutcome(event.next_round);
 
+            // this rotates the cache_
```

Comment:
ookkayy.. the comment seems confusing a little

Reply:
I added the comment because one hardly can deduce from the function name that it rotates the cache. And the fact that it is rotated is crucial here.